sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -66,45 +66,34 @@ export const environmentSetup: RecorderEnvironmentSetup = {
 };
 
 export function createRecordedCommunicationIdentityClient(
-  context: Context
+  context: Context,
+  withToken: boolean = false
 ): RecordedClient<CommunicationIdentityClient> {
   const recorder = record(context, environmentSetup);
 
-  return {
-    client: new CommunicationIdentityClient(env.COMMUNICATION_CONNECTION_STRING),
-    recorder
-  };
-}
-
-export function createRecordedCommunicationIdentityClientWithToken(
-  context: Context
-): RecordedClient<CommunicationIdentityClient> | undefined {
-  const recorder = record(context, environmentSetup);
-  let credential: TokenCredential;
-
-  if (isPlaybackMode()) {
-    credential = {
-      getToken: async (_scopes) => {
-        return { token: "testToken", expiresOnTimestamp: 11111 };
-      }
-    };
-
+  if (!withToken) {
     return {
-      client: new CommunicationIdentityClient(env.COMMUNICATION_ENDPOINT, credential),
+      client: new CommunicationIdentityClient(env.COMMUNICATION_CONNECTION_STRING),
       recorder
     };
   }
 
   try {
-    credential = new DefaultAzureCredential();
-  } catch {
-    return undefined;
-  }
+    let credential: TokenCredential = isPlaybackMode()
+      ? {
+          getToken: async (_scopes) => {
+            return { token: "testToken", expiresOnTimestamp: 11111 };
```

Comment:
what is the significant of using 11111 as timestamp here> may add a comment.

Reply:
Just a mock place holder. No significance to it

Reply:
I believe Josh just added a random number